themes/exegetic/layouts/index.html

```diff
@@ -1,13 +1,18 @@
 {{ define "header" }}
+    <a href="https://www.fathomdata.dev/" style="text-decoration: none;">
       <div class="full-page-banner-image d-flex align-items-center" style="background-image: url('img/exegetic-home-background.svg');">
-          <div class="container ">
+          <div class="container-fluid p-0">
             <div class="row align-items-center">
-              <img style="max-width: 100%" src="img/exegetic-banner-white.svg">
-              <div class="col-lg-12">
+              <img style="max-width: 70%; position: absolute; left: 16%;" src="img/exegetic-banner-white.svg">
+              <div class="col-lg-12 text-center" style="color: white; background-color: #9088ffc8; padding: 0.2% 0% 1% 0%;">
+                <h1 style="font-size: 3rem">We've rebranded!</h1>
+                <h2 style="font-size: 2rem">Visit our new website at fathomdata.dev!</h2>
+                <img src="img/fathom-logo.svg" style="height: 2rem;">
```

Comment:
@Axiematic rather than including the SVG in this repository, how about simply linking to the SVG that's hosted on the new site? You'd just put that URL here and then remove the local copy of the SVG?

Duplication of content should always be avoided. Why? Suppose that you make a change to the Fathom Data logo and update the SVG... this means that you need to make that update in two places rather than one.

Reply:
Ahh, sorry @datawookie I should've thought of this. Sorted.